src/components/Popup.js

```diff
@@ -9,6 +9,7 @@ export default class Popup {
     this._popupElement.classList.add("modal_opened");
     document.addEventListener("keydown", this._handleEscClose);
     document.addEventListener("mousedown", this._handleClickOutsideModal);
+    this.resetButtonText(".modal__button");
```

Comment:
`this.resetButtonText(".modal__button");` should be removed from `Popup` because the image modal doesn't have any button. 

There should **not** be any code in universal functions/classes that doesn’t suit every popup.